sklearn/decomposition/tests/test_incremental_pca.py

```diff
@@ -384,3 +384,16 @@ def test_incremental_pca_partial_fit_float_division():
 
     np.testing.assert_allclose(singular_vals_float_samples_seen,
                                singular_vals_int_samples_seen)
+
+
+def test_incremental_pca_fit_overflow_error():
```

Comment:
Could you please add an inline comment to explain that this is a non-regression problem for an oveflow problem on Windows with the URL of the issue:  https://github.com/scikit-learn/scikit-learn/issue/17693